### ISSUE TITLE
Flagged Attachment rule

### ISSUE THREAD

**Issue by warriordog** [warriordog/ModShark#42]
Create a new queued rule to scan media and non-media attachment. Initial flags should include pattern matching against alt text (caption / description) and filename. The following filters should be available:

* `IncludeLocal` - include local media (default true)
* `IncludeRemote` - include remote media (default true)
* `IncludeSuspendedUser` - include media by a suspended user (default true)
* `IncludeSilencedUser` - include media by a silenced user (default true)
* `IncludeBlockedInstance` - include media from a blocked (defederated) instance (default true)
* `IncludeSilencedInstance` - include media from a silenced (limited) instance (default true)

The rule should be built with extension in mind, as additional issues will expand the flags.
